### PR TITLE
[news] Add Final Review and carousel

### PR DESCRIPTION
**Pull request by fabiencastan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -157,6 +157,104 @@ <h1 class="text-center text-info" id="news">News</h1>
             </ul>
           </div>
         </div>
+        <div class="carousel slide" id="news-carousel" data-interval="5000" data-ride="carousel">
+          <div class="carousel-inner">
+            <div class="item active">
+              <img src="images/news/2016-08_grimstad_1.JPG">
+              <div class="carousel-caption">
+                <h2>Final POPART Review at Equippe AS Studio</h2>
+                <h3>Août 2016 - Grimstad (Norway)</h3>
```
Comment (simogasp) [inline index.html]: August

File: index.html [hunk 2]
```diff
@@ -157,6 +157,104 @@ <h1 class="text-center text-info" id="news">News</h1>
...
+              </div>
+            </div>
+            <div class="item">
+              <img src="images/news/2016-08_grimstad_2.JPG">
+              <div class="carousel-caption">
+                <h2>Final POPART Review at Equippe AS Studio</h2>
+                <h3>Août 2016 - Grimstad (Norway)</h3>
```
Comment (simogasp) [inline index.html]: August everywhere

File: index.html [hunk 3]
```diff
@@ -157,6 +157,104 @@ <h1 class="text-center text-info" id="news">News</h1>
...
+              <img src="images/news/2015-12-15_capdigital_3.JPG">
+              <div class="carousel-caption">
+                <h2>Intermediate POPART Review at Cap Digital</h2>
+                <h3>December 2015 - Paris (France)</h3>
+              </div>
+            </div>
+            <div class="item">
+              <img src="images/news/2015-10_immersed_gaming_1.JPG">
+              <div class="carousel-caption">
+                <h2>Immersed Gaming at MMSys</h2>
```
Comment (simogasp) [inline index.html:235]: Immersive?

Comment (fabiencastan) [reply index.html:235]: That's the title of the paper: http://dl.acm.org/citation.cfm?id=2910632

File: index.html (hunk 1 above)
Comment (fabiencastan) [reply index.html]: sure ;)

File: index.html [hunk 4]
```diff
@@ -155,6 +157,104 @@ <h1 class="text-center text-info" id="news">News</h1>
             </ul>
           </div>
         </div>
+        <div class="carousel slide" id="news-carousel" data-interval="5000" data-ride="carousel">
```
Comment (simogasp) [inline index.html:160]: i cannot use the mouse to navigate the slides (on local version), it only works with the arrow keys. Is it normal?

Comment (fabiencastan) [reply index.html:160]: No, I changed the name while commiting... and broke that.

Comment (fabiencastan) [reply index.html:160]: fixed

**Pull request merged**